drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -304,7 +304,6 @@ static int _get(netdev_t *netdev, netopt_t opt, void *val, size_t max_len)
     }
 
     uint8_t old_state = at86rf2xx_get_status(dev);
-    res = 0;
```

Comment:
Every possible path in the switch block below sets the res variable, so it is not necessary to set it here.